src/index.html

```diff
@@ -346,14 +361,21 @@ <h3 class="p-content__grid__parts__title">安心・安全</h3>
             </div>
             <div class="p-content__grid__parts">
               <h3 class="p-content__grid__parts__title">わかりやすい</h3>
-              <p class="p-content__grid__parts__text">当求人サイトは医療事務に特化しておりさらにこだわり検索や地域から検索<br>
-                といったご自身にピッタリマッチした求人を見つけることができる！</p>
+              <p class="p-content__grid__parts__text">
+                当求人サイトは医療事務に特化しておりさらに<br class="is-sp">
+                こだわり検索や地域から検索<br class="is-pc">
+                といった<br class="is-sp">ご自身にピッタリマッチした求人を見つけることができる！
+              </p>
```

Comment:
[must]

以下のように修正をお願いします！

```suggestion
              <p class="p-content__grid__parts__text">当求人サイトは医療事務に特化しておりさらに<br class="is-sp">こだわり検索や地域から検索<br class="is-pc">といったご自身に<br class="is-sp">ピッタリマッチした求人を見つけることができる！</p>
```

理由としては以下のようになります。

- is-sp の時に、「ご自身に」の後に改行されているため
- HTML 上で改行をしてしまうと、画面上ではスペースが1つ空いてしまうので、HTML上では、1行に収まるように記述する必要があるため（こちらに関しては他の箇所の記述も見直してみてください！）

Reply:
修正いたしました！
1070e0c8925cfa5e140e7b0845e417ff29f49479